server/routes/api.py

```diff
@@ -244,6 +244,56 @@ def post_comment(author_id: int, post_id: int) -> Response:
     return Response(status=httpStatus.OK)
 
 
+@bp.route("/authors/<int:author_id>/followers", methods=["GET"])
+def get_followers(author_id: int) -> Response:
```

Comment:
should we protect this endpoint with `login_required`?

Reply:
nope, we don't need to, the specs don't say nor imply we do